lib/wickedbase.pm

```diff
@@ -370,6 +372,25 @@ sub do_mutex {
     barrier_wait($barrier_name);
 }
 
+=head2 setup_vlan
+
+    setup_vlan($ip_type)
```

Comment:
shouldn´t it be $ip_alias?

Reply:
hm I would go actually another way around and rename a variable, when I was writing documentation I had idea in my head that variable should be `ip_type` because it holding value for `get_ip(type => $ip_type)` but I lost idea while writing code :(. WDYT ?

Reply:
+1 for $ip_type or just $type